tests/app/phase-tester-response/issue-responded/issue-responded.component.spec.ts

```diff
@@ -0,0 +1,45 @@
+import { IssueRespondedComponent } from '../../../../src/app/phase-tester-response/issue-responded/issue-responded.component';
+import { Issue, STATUS } from '../../../../src/app/core/models/issue.model';
+import { ISSUE_WITH_EMPTY_DESCRIPTION } from '../../../constants/githubissue.constants';
+import { IssueComment } from '../../../../src/app/core/models/comment.model';
+import { USER_JUNWEI } from '../../../constants/data.constants';
+
+describe('IssuesPendingComponent', () => {
+  describe('.ngOnInit()', () => {
+    const issuesRespondedComponent = new IssueRespondedComponent();
+    issuesRespondedComponent.ngOnInit();
+    const DUMMY_TEAM = USER_JUNWEI.team;
+    const DUMMY_COMMENT: IssueComment = {
+      id: 1,
+      description: 'This is some description of an issue comment',
+    };
+    let dummyIssue: Issue;
+
+    beforeEach(() => {
+      // generate a well-formed dummy-issue
+      dummyIssue = Issue.createPhaseTeamResponseIssue(
+        ISSUE_WITH_EMPTY_DESCRIPTION,
+        DUMMY_TEAM
+      );
+      dummyIssue.status = STATUS.Done;
```

Comment:
It would be better to assign its attributes within the tests itself for consistency. 
As a result, we can simply just assign the values within the tests like what #563 and #592 did ><

Reply:
Yea on hindsight I think assigning the attributes within the tests is more readable.
Will change :+1: